app/components/soda-form.js

```diff
@@ -1,4 +1,31 @@
 import Ember from 'ember';
+const {
+  set
+} = Ember;
 
 export default Ember.Component.extend({
-});
+  store: Ember.inject.service( 'store' ),
+  
+  isDiet: false,
```

Comment:
set on new model pls, set this as null so we know this component expects it to be sent